AppCenterDemoApp/Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -208,6 +208,16 @@ private IEnumerator SetEnabledCoroutine(bool enabled)
         DistributeEnabled.isOn = isDistributeEnabled.Result;
     }
 
+    public void SetCustomDialog(bool enable)
+    {
+        int isEnable = enable ? 1 : 0;
+        PlayerPrefs.SetInt(FlagCustomDialog, isEnable);
+        PlayerPrefs.Save();
+#if UNITY_ANDROID && !UNITY_EDITOR
```

Comment:
You can remove this. It's only if you need to access the preferences from native code.